main/system.c

```diff
@@ -4,8 +4,11 @@
 
 #include "i2c_master.h"
 #include "DS4432U.h"
+#include "TPS546.h"
```

Comment:
TPS546 should be astracted by the ne VCORE module introduced recently (so did DS4432 have been)